proto/osmosis/concentrated-liquidity/genesis.proto

```diff
@@ -48,14 +48,19 @@ message PoolData {
       [ (gogoproto.nullable) = false ];
 }
 
+message PositionData {
+  Position position = 1;
+  uint64 lock_id = 2 [ (gogoproto.moretags) = "yaml:\"lock_id\"" ];
+}
+
 // GenesisState defines the concentrated liquidity module's genesis state.
 message GenesisState {
   // params are all the parameters of the module
   Params params = 1 [ (gogoproto.nullable) = false ];
   // pool data containining serialized pool struct and ticks.
   repeated PoolData pool_data = 2 [ (gogoproto.nullable) = false ];
 
-  repeated Position positions = 3 [ (gogoproto.nullable) = false ];
+  repeated PositionData position_data = 3 [ (gogoproto.nullable) = false ];
```

Comment:
So, it seems like the only reason we need to do this is because lock_id is the only externally linked data in all of the KVStore pairs. Maybe it makes sense for us to include underlying lock ID in the position struct after all...

Thoughts @p0mvn ?

Reply:
Why did we remove it from the second index in the first place? Was it the fact that some position IDs don't have a lock id associated with them? If that's the case, I think the current approach is better.

Reply:
I removed it from the denom because of the complexity it introduced when attempting to create intermediary accounts with the base denom rather than the full denom (it would try and gather all locks with the base denom rather the full one). Also, removing it reduced much of the custom lock logic that was added previously

Reply:
will proceed with current approach and we can change anything that comes up in the future